src/hub/hubNavigation.ts

```diff
@@ -0,0 +1,17 @@
+import { env, Uri } from "vscode";
+import { configuration } from "../binary/requests/requests";
+import { StateType } from "../globals/consts";
+import openHub from "./openHub";
+
+export default async function open(view: string): Promise<void> {
```

Comment:
`default exported function != file name`
generally speaking - the name open/hubNavigation is not clear